packages/accounts/useraccounts_templates.coffee

```diff
@@ -0,0 +1,7 @@
+Template.signIn.events
+  'click #at-signUp' : ->
+    FlowRouter.go "/signUp"
```

Comment:
This package shouldn't depend on FlowRouter directly. We could add something to the 'path' package and rename it.